src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -431,25 +431,19 @@ private MLNodeAutoCompletionResponse GetMLNodeAutocompleteResults(string request
             MLNodeAutoCompletionResponse results = null;
             var authProvider = dynamoViewModel.Model.AuthenticationManager.AuthProvider;
 
-            if (authProvider is IOAuth2AuthProvider oauth2AuthProvider)
+            if (authProvider is IOAuth2AuthProvider oauth2AuthProvider && authProvider is IOAuth2AccessTokenProvider tokenprovider)
```

Comment:
@reddyashish you may want to use this interface to get a dynamo app access token from our auth provider - see below.